src/components/stores/event-store.ts

```diff
@@ -1,43 +0,0 @@
-import { StoredEvent } from "../../models/events/event";
```

Comment:
I may rebase this PR to split out these file removals (which have since been committed as 7018468).